packages/loader/container-loader/src/container.ts

```diff
@@ -587,14 +587,22 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                     dmLastMsqSeqNumber: () => this.deltaManager?.lastMessage?.sequenceNumber,
                     dmLastMsqSeqTimestamp: () => this.deltaManager?.lastMessage?.timestamp,
                     dmLastMsqSeqClientId: () => this.deltaManager?.lastMessage?.clientId,
-                    connectionState: () => ConnectionState[this.connectionState],
```

Comment:
Is this intentional?

Reply:
its a duplicated value, we already log connextion state in the above `all` block, so it will log on all events

Reply:
Didn't see the `all` block